VAMobile/e2e/tests/Messages.e2e.ts

```diff
@@ -355,7 +355,7 @@ describe('Messages Screen', () => {
     await element(by.text(CommonE2eIdConstants.VETERAN_CRISIS_LINE_BTN_TEXT)).tap()
     await expect(element(by.text('Veterans Crisis Line'))).toExist()
     await element(by.text('Done')).tap()
-  })
+  // })
```

Comment:
Is this comment intentional?

Reply:
good catch

I think this may have been when Rachael was trying to fix them